lib/schemas/HAR/timings.json

```diff
@@ -7,18 +7,10 @@
     "receive"
   ],
   "properties": {
-    "dns": {
-      "type": "number",
-      "min": -1
-    },
     "connect": {
```

Comment:
In the specification, this is mentioned as optional and to be removed but it's not removed here.

Reply:
Thanks, missed this! Will remove them